### ISSUE TITLE
EPANET 2.2 support

### ISSUE THREAD

**Issue by kalebsm** [USEPA/WNTR#143]
Hello all,
I was wondering with the recent release of EPANET 2.2 GUI now with pressure driven demand (https://www.epa.gov/water-research/epanet) if anybody was working on updating WNTR to work on the dynamic link libraries, etc of EPANET 2.2. Specifically, I have a code that uses run_sim in the wntr.sim.epanet module, and it would be nice to use the same process to run on EPANET 2.2 with pressure driven demand.

Thanks,
Kaleb Smith

**Comment by dbhart** [comment]
Hi Kaleb. EPANET 2.2 support is on the process of being pushed up into the release branch. In the meantime, the changes you are requesting have already been implemented in the epanet22 branch in my fork (dbhart). Documentation has been updated, and testing is passing. If you run across any bugs, please let is know.

- Dave Hart, Sent from my iPhone

On Jun 14, 2020, at 2:41 PM, kalebsm <notifications@github.com> wrote:

﻿

Hello all,
I was wondering with the recent release of EPANET 2.2 GUI now with pressure driven demand (https://www.epa.gov/water-research/epanet) if anybody was working on updating WNTR to work on the dynamic link libraries, etc of EPANET 2.2. Specifically, I have a code that uses run_sim in the wntr.sim.epanet module, and it would be nice to use the same process to run on EPANET 2.2 with pressure driven demand.

Thanks,
Kaleb Smith

**Comment by dbhart** [comment]
Also as a heads-up. In the upcoming epanet22 integrated release of wntr, we have renamed some of the options within the WaterNetworkOptions class to be consistent with epanet22 as well as adding in the binaries. A “version=2.2” option has been added to the EpanetSimulator.run_sim() command, but the default will be EPANET 2.2 in the future (and will be on my (dbhart) fork/branch within a few minutes).

Hope this helps,

Dave H.
